database-plugins/src/main/java/io/cdap/plugin/db/batch/source/DBSource.java

```diff
@@ -189,6 +192,11 @@ public void prepareRun(BatchSourceContext context) {
                               new SourceInputFormatProvider(DataDrivenETLDBInputFormat.class, hConf)));
 
     emitLineage(context);
```

Comment:
can modify this private method instead of introducing new method, otherwise we are emitting the lineage twice now in source.

Reply:
Added getLineageRecorder method and used the existing emitLineage method.